svelte-responsive-layout/src/App.svelte

```diff
@@ -0,0 +1,184 @@
+<script>
+  import { onMount, onDestroy } from 'svelte';
+  import TopNav from './TopNav.svelte';
+  import ContentButton from './ContentButton.svelte';
+  import ResponsiveImage from './ResponsiveImage.svelte';
+
+  let detached = false;
+
+  function handleScrollChange({ target }) {
+    detached = !!window.scrollY;
+  }
+
+  onMount(() => document.addEventListener('scroll', handleScrollChange));
+
+  onDestroy(() => document.removeEventListener('scroll', handleScrollChange));
+
+  function handleDiscoverButtonClick() {
+    location.href = "https://github.com/sveltejs/svelte";
+  }
+</script>
+
+<!--
+  There are some troubles with PostCss here.
+  We cannot use autocomplete because of some issues with svelte-vscode
+-->
+<style type="text/postcss">
+  @custom-media --big-viewport (min-width: 640px);
+
+  :root {
+    --main-color: #fff;
+    --secondary-color: #45557e;
+
+    --header-height: 10vh;
+  }
+
+  .page-layout {
+    max-width: 960px;
+    margin: 0 auto;
+    padding: 0 20px;
```

Comment:
Based on 62.5% as done previously, now you just need to convert all 20px to be /10. ex here: `padding: 0 2rem`